### PR TITLE
Animated paragraphs

### PR DESCRIPTION
**Pull request by piotrkramarz**
Add animated paragraphs to sections. Please advise if you have any remarks.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,9 +9,13 @@
     <link rel="stylesheet" href="assets/styles/reset.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
     <link rel="stylesheet" href="assets/styles/general.css">
+    <link rel="stylesheet" type="text/css" href="http://cdn.bootcss.com/animate.css/3.1.0/animate.css">
+
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="http://code.jquery.com/jquery-1.11.0.js"></script>
```
Comment (cytrowski) [inline index.html]: Uważaj, bo masz tutaj drugi raz załączone jQuery na stronie

Comment (wojtrawi) [inline index.html]: jQuery is already included (see above).

**Review by cytrowski — DISMISSED** [review]
Usuń nadmiarowe załączenia jQuery

**Review by wojtrawi — CHANGES_REQUESTED** [review]

**Comment by piotrkramarz** [comment]
Second jQuery link deleted. Please accept pull request.

**Review by wojtrawi — DISMISSED** [review]

**Review by Czapelxd — APPROVED** [review]
look legit

**Pull request merged**